libs/ui/popover/package.json

```diff
@@ -24,7 +24,7 @@
     "@terminus/design-tokens": "^1.0.10",
     "@terminus/ngx-tools": "^8.0.5",
     "@terminus/ui-utilities": "^0.0.1",
-    "popper.js": "^1.16.0",
+    "@popperjs/core": "2.0.0",
```

Comment:
With `^`, it gets the version of `2.4.0` which unfortunately breaks some styles. So for now, we're restricting the version.